sample/build.gradle.kts

```diff
@@ -20,8 +65,19 @@ android {
         resValues = true
         viewBinding = true
     }
-}
 
-dependencies {
-    implementation(project(":chart"))
+    lintOptions {
+        // Good habits for a real app, but trying to keep the sample project minimal.
+        disable += "AllowBackup"
+        disable += "MissingApplicationIcon"
+    }
+
+    sourceSets {
+        val main by getting {
+            // FIXME: This feels like I'm working around some other misconfiguration.
+            java.srcDirs("src/androidMain/java")
+            res.srcDirs("src/androidMain/res")
```

Comment:
```suggestion
```

I've found these aren't needed. Without them you don't get the nice blue folder icon (for `java`):

![Screen Shot 2021-04-13 at 11 47 17 AM](https://user-images.githubusercontent.com/98017/114605745-40e48680-9c4f-11eb-81e6-f7837780b7af.png)

...but, if you rename your `java` folder to `kotlin`, then you still get your blue folder without these lines:

![Screen Shot 2021-04-13 at 11 48 32 AM](https://user-images.githubusercontent.com/98017/114605815-52c62980-9c4f-11eb-8be4-623edda043d9.png)

In my experience you still need the `manifest.srcFile` though.

Reply:
Ah! Perfect. That's the "other misconfiguration" I was talking about